src/Lykke.Payments.Link4Pay.Domain/Services/BooleanJsonConverter.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using Newtonsoft.Json;
+
+namespace Lykke.Payments.Link4Pay.Domain.Services
+{
+    public class BooleanJsonConverter : JsonConverter
```

Comment:
do they really use string instead of bool values in json?

Reply:
yes they do (